examples/jss/src/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { render } from 'react-dom';
+import jss from 'jss';
+import preset from 'jss-preset-default';
+
+jss.setup(preset());
+
+const globalStyles = {
+  // Global styles
+  html: {
+    color: '#292f33',
+    fontFamily: '"Helvetica Neue", Helvetica, Arial, sans-serif',
+    fontSize: '14px',
+    lineHeight: '1.3125',
+  },
+  a: {
+    textDecoration: 'none',
+    color: '#1da1f2',
+  },
+  svg: {
+    fill: 'currentColor',
+    height: '1.25em',
+  },
+  '@media screen and (min-width: 360px)': {
```

Comment:
to show the beauty of js, you could put all media queries into a separate file and then use them like this:

```
...

[screenSizes.xs]: {
}
...
```